docs/generators/java.md

```diff
@@ -15,8 +15,8 @@ sidebar_label: java
 |apiPackage|package for generated api classes| |null|
 |invokerPackage|root package for generated code| |null|
 |groupId|groupId in generated pom.xml| |null|
-|artifactId|artifactId in generated pom.xml| |null|
-|artifactVersion|artifact version in generated pom.xml| |null|
+|artifactId|artifactId in generated pom.xml. This also becomes part of the generated library's filename (default value 'openapi-java-client')| |null|
```

Comment:
@intelliot thanks for the PR. This file is auto-generated. Please update https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java#L67 instead and then run `./bin/utils/ensure-up-to-date` to update the auto-generated doc.